shared/chat/conversation/messages/header/container.js

```diff
@@ -3,16 +3,26 @@ import * as Constants from '../../../../constants/chat'
 import Header from '.'
 import {compose} from 'recompose'
 import {connect} from 'react-redux'
+import {CommonConversationMembersType} from '../../../../constants/types/flow-types-chat'
 
 import type {TypedState} from '../../../../constants/reducer'
 import type {OwnProps} from './container'
 
 const mapStateToProps = (state: TypedState, {messageKey}: OwnProps) => {
   const conversationState = Constants.getSelectedConversationStates(state)
   const moreToLoad = conversationState && conversationState.get('moreToLoad')
+  const inbox = Constants.getSelectedInbox(state)
+
+  // If it's a multi-user chat that isn't a team, offer to make a new team.
+  const showTeamOffer =
+    inbox &&
+    inbox.info &&
+    inbox.info.membersType !== CommonConversationMembersType.team &&
+    inbox.get('participants').size > 2
```

Comment:
i think we always want to use count(). you can use setSize() to change the size of the list but that'll include non-items